linux/usage/users.md

```diff
@@ -30,25 +30,21 @@ Upon creating a new user, the contents of `/etc/skel/` will be copied to the new
 
 ## Sudoers
 
-The default `pi` user on Raspbian is a sudoer. This gives the ability to run commands as root when preceded by `sudo`, and to switch to the root user with `sudo su`.
+The default `pi` user on Raspbian is a member of the `sudo` group. This gives the ability to run commands as root when preceded by `sudo`, and to switch to the root user with `sudo su`.
 
-To add a new user to sudoers, type `sudo visudo` (from a sudoer user) and find the line `root    ALL=(ALL:ALL) ALL`, under the commented header `# User privilege specification`. Copy this line and switch from `root` to the username. To allow passwordless root access, change to `NOPASSWD: ALL`. The example below gives the user `bob` passwordless sudo access:
+To add a new user to the `sudo` group, use the `adduser` command:
 
 ```bash
-# User privilege specification
-root  ALL=(ALL:ALL) ALL
-bob   ALL = NOPASSWD: ALL
+sudo adduser bob sudo
 ```
```

Comment:
@JamesH65 
Would have been an idea to mention as well how to add an existing user to the sudo group:
```
To add an existing user to the `sudo` group, use the `usermod` command:

\```bash
sudo usermod -aG sudo bob
\```
Note the reversed order of group and user compared to `adduser`.
```